tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-integration-test.php

```
@@ -178,5 +178,43 @@ public function test_constants() {
+	}
+
+	/**
+	 * Test is_merchant_enabled returns false when option is not set.
+	 *
+	 * @return void
+	 */
+	public function test_is_merchant_enabled_default_false() {
+		delete_option( \WC_Stripe_Agentic_Commerce_Integration::ENABLED_OPTION );
+
+		$this->assertFalse( \WC_Stripe_Agentic_Commerce_Integration::is_merchant_enabled() );
+	}
```

PHPMD (2.15.0)

[error] The method test_is_merchant_enabled_default_false is not named in camelCase. (undefined)

(CamelCaseMethodName)